Sources/SnapshotTesting/Snapshotting/URLRequest.swift

```diff
@@ -46,17 +46,18 @@ extension Snapshotting where Value == URLRequest, Format == String {
     var components = ["curl"]
 
     // HTTP Method
-    let httpMethod = request.httpMethod!
-    switch httpMethod {
-    case "GET": break
-    case "HEAD": components.append("--head")
-    default: components.append("--request \(httpMethod)")
+    if let httpMethod = request.httpMethod {
+      switch httpMethod {
+      case "GET": break
+      case "HEAD": components.append("--head")
+      default: components.append("--request \(httpMethod)")
+      }
```

Comment:
Let's leave these lines as the original:

```suggestion
    let httpMethod = request.httpMethod!
    switch httpMethod {
    case "GET": break
    case "HEAD": components.append("--head")
    default: components.append("--request \(httpMethod)")```
```

I'm not aware of any time that `httpMethod` can actually be `nil`, see for example:

```swift
var request = URLRequest(url: .init(string: "http://www.apple.com")!)
request.httpMethod // "GET"
request.httpMethod = nil
request.httpMethod // "GET"
```

I'm guessing this is `nil` due to an oversight during early Swift optionality audits, and so rather assume its non-`nil` until there's reproduction steps to show how it can be made `nil`.